.github/actions/reusable-python-setup_poetry/action.yml

```diff
@@ -9,18 +9,13 @@ inputs:
     description: 'Runner OS'
     required: true
     type: string
-  extra-repository:
-    description: 'Boolean to indicate if extra repository is needed'
-    required: false
-    type: boolean
-    default: false
-  repo-user:
-    description: 'Repository username'
+  virtual-repo-names:
+    description: 'Matrix of virtual repository names to resolve (eg. ["public-dev-pypi"])'
     required: false
     type: string
-  repo-token:
-    description: 'Repository token'
-    required: false
+  JFROG_ARTIFACTORY_TOKEN:
```

Comment:
**quibble (not-blocking):** not the usual naming convention? `lower-case`

Reply:
For some reason, secrets are mostly written in capital letters. This is the rule we follow in all our workflows.